forms/models/form.js

```diff
@@ -17,9 +17,17 @@ define(function (require) {
   var Form;
   var invalidWrapperFn, isSubForm;
 
+  function isVisible(elementModel){
+    if ( !elementModel.has('hidden')){
```

Comment:
Is this check faster or different somehow to just the `return` line all by itself?